### DISCUSSION TITLE
Sync between macs

### DISCUSSION THREAD

**Discussion by serbanveres** [mikekazakov/nimble-commander#453] [Q&A]
I love nimble commander, think it’s the closest mac app to total commander. I’d love to be able to sync the settings between my two macs. Can you please tell me in which file do you save the configurations? I’m especially interested in syncing the ftp connections. I’d copy the file on icloud and then create a symlink. Just that I don’t know where that file is…

**Comment by mikekazakov** [comment]
Hi, please take a look at https://github.com/mikekazakov/nimble-commander/blob/main/Docs/Help.md#frequently-asked-questions:
```
Main configuration files (managed by NC):
~/Library/Application Support/Nimble Commander/Config
Volatile state file (managed by NC):
~/Library/Application Support/Nimble Commander/State
```
Network connections are stored specifically in `~/Library/Application Support/Nimble Commander/Config/NetworkConnections.json`.
